src/shared/common/forms/mapped-display.jsx

```diff
@@ -10,26 +10,35 @@ class MappedDisplay extends Component {
     render() {
         const {
             choices,
+            component,
             label,
             value
         } = this.props;
         const mappings = new Map(choices.map(choice => [choice.value, choice.label]));
-        return (
-            <span className='govuk-body full-width'><strong>{label}: </strong>{mappings.get(value) ? mappings.get(value) : value}</span>
-        );
+        if (component === 'checkbox') {
+            return (
+                <span className='govuk-body full-width'><strong>{choices[0].label}: </strong><input className="bigger" type="checkbox" disabled="disabled" checked="checked"/></span>
```

Comment:
Two questions on this here:

- Are we sure the label field will always exist in the **first** element in the choices variable? Conscious of magic numbers.
- Has this approach been run past designer/assessed for accessibility? A disabled input field used solely for presentational purpose has a bit of a smell, potential that screen readers could completely ignore that an element is even there: https://a11y-101.com/development/aria-disabled

Reply:
The label field is there in all of the existing checkbox choices definitions. It's all magic!

The problem with the way checkboxes are implemented on DECS is that they have all sorts of values. See note on the ticket. 
The guidance didn't help!

I could set it to the text "Selected"?

I'll have a chat with Ash.

Reply:
@robertdeniszczyc2 I spoke to Ash. He has suggested the use of 'Yes' instead of a checkbox.